### PR TITLE
Add Aevilok; Craftsman

### PR DESCRIPTION
**Pull request by hakr14**

### CODE REVIEW

File: subclass/Aevilok; Craftsman.json [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+					"subclassFeature": "Mundane Creations|Rogue||Craftsman|roguecraft|3"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Junkyard Survival|Rogue||Craftsman|roguecraft|3"
+				}
+			]
+		},
+		{
+			"type": "entries",
```
Comment (TheGiddyLimit) [inline subclass/Aevilok; Craftsman.json:59]: `"type": "entries",` isn't allowed on the root of a `subclassFeature`, as it is implicit. Delete this line

File: subclass/Aevilok; Craftsman.json [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+			"page": 1,
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Craftsman",
+			"subclassSource": "roguecraft",
+			"level": 3,
+			"header": 1
+		},
+		{
+			"type": "entries",
```
Comment (TheGiddyLimit) [inline subclass/Aevilok; Craftsman.json:90]: ...and this one...

File: subclass/Aevilok; Craftsman.json [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+			"page": 1,
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Craftsman",
+			"subclassSource": "roguecraft",
+			"level": 3,
+			"header": 2
+		},
+		{
+			"type": "entries",
```
Comment (TheGiddyLimit) [inline subclass/Aevilok; Craftsman.json:105]: ...and this one...

File: subclass/Aevilok; Craftsman.json [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
```
Comment (TheGiddyLimit) [inline subclass/Aevilok; Craftsman.json:148]: ...and this one.

**Review by TheGiddyLimit — CHANGES_REQUESTED** [review]

File: subclass/Aevilok; Craftsman.json (hunk 1 above)
Comment (hakr14) [reply subclass/Aevilok; Craftsman.json]: Well that's... odd. I have another homebrew that uses `"type": "entries"` for the subfeatures, and that works... The file worked fine when I uploaded it to homebrew manager manually too...
Well, in either case, the issue here is fixed. Thanks!

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**